wbooks/src/app/components/BookDetailItem/styles.tsx

```diff
@@ -0,0 +1,39 @@
+import { StyleSheet } from 'react-native';
+import Colors from '@constants/colors';
+import Fonts from '@constants/fonts';
+
+const styles = StyleSheet.create({
+  container: {
+    borderRadius: 5,
+    minHeight: 270,
```

Comment:
No, never use fixed height and sizes in general=> This is only for images and specific cases. Use just margin and padding for spacing